contrib/aktualizr-toml-update

```diff
@@ -4,16 +4,25 @@
 # It copies the toml file to /etc/sota/conf.d/ and will restart aktualizr-lite
 
 [ -z "$CONFIG_FILE" ] && (echo "No CONFIG_FILE specified"; exit 1)
-mkdir -p -m 0700 /etc/sota/conf.d/
-cp $CONFIG_FILE /etc/sota/conf.d/
 
-# If we take control of tags/apps, then we should try and remove from sota.toml
-if grep -q "^\s*tags\s*=" $CONFIG_FILE 2>/dev/null ; then
-	sed -e '/^\s*tags\s*=/ s/^#*/# MANAGED BY FIOCONFIG: /' -i /var/sota/sota.toml
-fi
+if [ -f $CONFIG_FILE ] ; then
+	mkdir -p -m 0700 /etc/sota/conf.d/
```

Comment:
both the "if" and "else" require /etc/sota/conf.d to exist. I think you need the `mkdir` outside of this?

Reply:
otherwise - i'm ready +1 after a `git squash`.

Reply:
An else doesn't need that directory, I've intentionally used a `rm -f` flag which ignores errors produced by inexistent file/folder.

Reply:
Note also that github will do a squash itself (there is a "Squash and Merge" options which does it nice).

Reply:
Sometimes github squash butchers your commit message. It also can't gpg sign your commits. No biggie for this. Merge at your leisure